recipes/gromacs/build.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/bash
+
+# gromacs > 2021 requires a non default OSX version
+if [ "$(uname)" = 'Darwin' ] ; then
+    export MACOSX_DEPLOYMENT_TARGET=10.13 
+fi
+
+
+mkdir build
+cd build
+
+## See INSTALL of gromacs distro
+for ARCH in SSE2 AVX_256 AVX2_256; do
+  cmake_args=(
+    -DSHARED_LIBS_DEFAULT=ON
+    -DBUILD_SHARED_LIBS=ON
+    -DGMX_PREFER_STATIC_LIBS=NO
+    -DGMX_BUILD_OWN_FFTW=OFF
+    -DGMX_DEFAULT_SUFFIX=ON
+    # Tests are not currently run, so do not download them
+    # -DREGRESSIONTEST_DOWNLOAD=ON
+    -DCMAKE_PREFIX_PATH="${PREFIX}"
+    -DGMX_INSTALL_PREFIX="${PREFIX}"
+    -DCMAKE_INSTALL_PREFIX="${PREFIX}"
+    -DGMX_SIMD="${ARCH}"
+    -DCMAKE_INSTALL_BINDIR="bin.${ARCH}"
+    -DCMAKE_INSTALL_LIBDIR="lib.${ARCH}"
+    -DGMX_VERSION_STRING_OF_FORK="conda-forge"
+  )
+  # OpenCL header on Mac is not recognized by GROMACS
+  if [ "$(uname)" != 'Darwin' ] ; then
+      cmake_args+=(-DGMX_GPU=OpenCL)
+  fi
+  if [[ "${mpi}" == "nompi" ]]; then
+      cmake_args+=(-DGMX_MPI=OFF -DGMX_THREAD_MPI=ON)
+  else
+      cmake_args+=(-DGMX_MPI=ON)
+  fi
+  cmake .. "${cmake_args[@]}"
+  make -j "${CPU_COUNT}"
+  make install
+done
+
+
+#
+# Build the program to identify number of AVX512 FMA units
+# This will only be executed on AVX-512-capable hosts. If there
+# are dual AVX-512 FMA units, it will be faster to use AVX-512 SIMD, but if
+# there's only a single one we prefer AVX2_256 SIMD instead.
+#
+
+#${CXX} -O3 -mavx512f -std=c++11 \
+#-DGMX_IDENTIFY_AVX512_FMA_UNITS_STANDALONE=1 \
+#-DGMX_X86_GCC_INLINE_ASM=1 \
+#-DSIMD_AVX_512_CXX_SUPPORTED=1 \
+#-o ${PREFIX}/bin.AVX_512/identifyavx512fmaunits \
+#${SRC_DIR}/src/gromacs/hardware/identifyavx512fmaunits.cpp
+## Create wrapper and activation scripts
+## Variable declaration from MPI script fewer changes if left in.
+
+
+if [ "${mpi}" = 'nompi' ] ; then
+    gmx='gmx'
+else
+    gmx='gmx_mpi'
+fi
+
+mkdir -p "${PREFIX}/etc/conda/activate.d"
+mkdir -p "${PREFIX}/etc/conda/deactivate.d"
+touch "${PREFIX}/bin/${gmx}"
+chmod +x "${PREFIX}/bin/${gmx}"
+
+# Copy the deactivate scripts for conda
+cp gromacs_deactivate.sh "${PREFIX}/etc/conda/deactivate.d"
```

Comment:
Perhaps using the `cat`-based approach below is one that avoids having to learn how to access the directory from which this script was read, because that need not relate to the current working directory at all.

Reply:
Yes, I was just thinking that was (another) benefit of using the `cat` approach.

The other option would be to put the `cp` command in the yaml build file - as we did for the prismatic feedstock: https://github.com/conda-forge/prismatic_split-feedstock/blob/a41dd1ef0330cdfb7e09aced025a64822ed1df50/recipe/meta.yaml#L152-L155

Reply:
Well, not quite the same, but we could probably have some commands to copy the deactivate scripts across.

Reply:
Ahh, no - we could use this instead:
```
cp "${RECIPE_DIR}/gromacs_deactivate.sh" "${PREFIX}/etc/conda/deactivate.d/"
```
That should work.

Reply:
Thanks - this worked.